cmd/validator/validator.go

```diff
@@ -17,6 +17,8 @@ optional flags:
     	Subdirectories to exclude when searching for configuration files
   -exclude-file-types string
     	A comma separated list of file types to ignore
+  -output
+  		Destination of a file to outputting results
```

Comment:
Indent is off here. Ensure it aligns with the rest of the descriptions please.

Reply:
fixed it. Thanks.